packages/bsky/src/hydration/actor.ts

```diff
@@ -167,6 +189,13 @@ export class ActorHydrator {
         blockingByList: parseString(rels.blockingByList),
         following: parseString(rels.following),
         followedBy: parseString(rels.followedBy),
+        knownFollowers:
+          knownFollowersDids?.length > 0
```

Comment:
maybe to just make this a little more clear, how about `knownFollowersDids && knownFollowersDids?.length > 0`?

that way we're not comparing `undefined > 0`